draft-ietf-httpbis-http2bis.xml

```diff
@@ -606,6 +600,49 @@ HTTP Frame {
           connection with a <xref target="ConnectionErrorHandler">connection error</xref> of type
           <xref target="COMPRESSION_ERROR" format="none">COMPRESSION_ERROR</xref> if it does not decompress a field block.
         </t>
+        <t>
+          A decoding error in a field block MUST be treated as a <xref
+          target="ConnectionErrorHandler">connection error</xref> of type <xref
+          target="COMPRESSION_ERROR" format="none">COMPRESSION_ERROR</xref>.
+        </t>
+
+        <section anchor="dynamic-table">
+          <name>Compression State</name>
+          <t>
+            Field compression is stateful.  One compression context and one decompression context
+            are used for the entire connection.  <xref target="COMPRESSION" section="4"/> defines
+            the dynamic table, which is the primary state that is used for field compression.
+          </t>
+          <t>
+            The dynamic table has a maximum size that is set by a decoder using the
+            SETTINGS_HEADER_TABLE_SIZE setting; see <xref target="SettingValues"/>.  Any change to
+            the maximum value set by the decoder takes effect when the encoder <xref
+            target="SettingsSync">acknowledges settings</xref>.  The encoder can set the dynamic
+            table to any size up to the maximum value set by the decoder.  The encoder declares the
+            size of the dynamic table with a Dynamic Table Size Update instruction (<xref
+            target="COMPRESSION" section="6.3"/>).  The encoder at both client and server is
+            initialized with a dynamic table size of 4,096 bytes, the initial value of the
+            SETTINGS_HEADER_TABLE_SIZE setting.
+          </t>
+          <t>
+            If a change to SETTINGS_HEADER_TABLE_SIZE reduces the maximum below the current size of
```

Comment:
```suggestion
            Once an endpoint acknowledges a change to SETTINGS_HEADER_TABLE_SIZE reducing the maximum below the current size of
```
I understand that the timing is clarified in the paragraph right above, but perhaps it might not hurt to clarify that here as well.